### ISSUE TITLE
Keep ontology and profile URLs stable

### ISSUE THREAD

**Issue by VladimirAlexiev** [Sveino/Inst4CIM-KG#57]
The URLs that @griddigit-ci cited in https://github.com/Sveino/Inst4CIM-KG/issues/56
(http://cim.ucaiug.io/grid/Dynamics/1.0, http://cim.ucaiug.io/grid/StandardInterconnections/1.0)
made me think about this:

The latest version of an ontology and profile should always be available at the same URL.
- Keeping ontology and profile URLs stable makes life much easier for ontology users (both data providers and consumers).
- It allows data conforming to older releases to coexist peacefully with data conforming to newer releases
- Usually ontology versions are backward compatible (i.e. the newer version extends the older version).
  - Since CIM UML is developed through a rigorous process, I hope that the meaning of terms doesn't change between releases (is this true?)
  - You should never delete terms: only mark them `owl:deprecated`.
- An ontology should also have a versioned URL and we should keep and publish older versions:
but instance data is recommended to use un-versioned URLs.

**Comment by VladimirAlexiev** [comment]
ENTSO-E document [RDF-Syntax User Guide v1.0](https://www.entsoe.eu/Documents/CIM_documents/Grid_Model_CIM/RDF-SyntaxUserGuide_v1-0.pdf) (2024-01-17) section 3. "Combining different CIM versions" describes difficulties related to exchange of semantic data that uses versioned terms, and concludes:
> Starting with CIM18, the CIM international standard development community agreed to keep the URI of the canonical CIM stable between different versions of CIM. This means that if a class is defined in CIM vocabulary its URI will not change. Semantic versioning is applied on profile level and different packages in CIM in order to be able to describe and explain CIM evolution.

**Comment by Sveino** [comment]
Yes, after a lot of work we managed to get the community to agree to use persistent URLs. We are now in the very difficult transition period. We also need to manged to implement solution that support that the URL should without version resolves to the current version. We would also need to implement support for future version (alfa version) for testing.
